content/en/docs/concepts/security/application-security-checklist.md

```diff
@@ -0,0 +1,84 @@
+- [ ] Appropriate Pod Security Standard policies are enforced to all pods in the application namespace.
+
+### Container Security Context
+- [ ] Disable privilege escalations using `allowPrivilegeEscalation: false`.
+- [ ] Configure the root filesystem to be read-only with `readOnlyRootFilesystem: true`.
+- [ ] Avoid running privileged containers with `privileged: false`.
+- [ ] Drop all capabilities from the containers, and add back only specific ones that are needed for operation of the container.
+
+### Role Based Access Control (RBAC)
+- [ ] Permissions such as `create`, `patch`, `update` and `delete` should be only granted if necessary.
+- [ ] Avoid creating RBAC permissions to create or update roles which can lead to [privilege escalation](/docs/reference/access-authn-authz/rbac/#privilege-escalation-prevention-and-bootstrapping).
+- [ ] Review bindings for the `system:unauthenticated` group and remove them where possible, as this gives access to anyone who can contact the API server at a network level.
+
+The `create`, `update` and `delete` should be used judicously. The `patch` permission can [allow users to update labels on the namespace or deployments](/docs/concepts/security/rbac-good-practices/#namespace-modification) which can increase the attack surface.
+
+### Image security
+- [ ] Using a image scanning tool the scan an image before deploying containers in the Kubernetes cluster.
+- [ ] Use container signing to validate the container image signature before deploying to the Kubernetes cluster.
+
+### Network Policies
+- [ ] Configure network policies to only allow expected ingress and egress traffic from the pods.
+
+## Advanced Security Hardening
+This section of this guide covers some advanced security hardening points which might be valuable based on different Kubernetes environment setup
+
+### Linux container security
+Configure {{< glossary_tooltip text="Security Context" term_id="Security-Context" >}} for the pod-container.
+- [ ] [Set appropriate Seccomp profiles.](/docs/tasks/configure-pod-container/security-context/#set-the-seccomp-profile-for-a-container)
+- [ ] [Configure appropriate AppArmor policies.](/docs/tutorials/security/apparmor/)
+- [ ] [Assign appropriate SELinux labels to the contianers.](/docs/tasks/configure-pod-container/security-context/#assign-selinux-labels-to-a-container)
+
+### Runtime Classes
+- [ ] Configure appropriate runtime classes for containers.
+
+Some containers may require a different isolation level from what is provided by the default runtime of the cluster. `runtimeClassname` can be used in a podspec to define a differnt runtime class.
+
+For sensitive workloads consider using Kernel emulation tools like [gVisor](https://gvisor.dev/docs/) or [kata-containers](https://katacontainers.io/).
+
+In high trust environments, consider using [confidential virutal machines](/blog/2023/07/06/confidential-kubernetes/) to improve cluster security even further.
```

Comment:
```suggestion
In high trust environments, consider using [confidential virtual machines](/blog/2023/07/06/confidential-kubernetes/) to improve cluster security even further.
```

Watch out for linking to non-evergreen blog articles (anything that gets marked as out of date once at least a year old).